resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
I think what we need is some string replacement in the release process e.q. when version `2.1` is released a placeholder in the files (like `RELEASE`) get's replaced by the actual value.